MMTabBarView/MMTabBarView/MMTabBarView.h

```diff
@@ -91,7 +91,7 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  *  Delegate
  */
-@property (weak)   IBOutlet id <MMTabBarViewDelegate> delegate;
+@property (nullable, strong)   IBOutlet id <MMTabBarViewDelegate> delegate;
```

Comment:
This will cause retain cycles

Reply:
https://github.com/ViennaRSS/MMTabBarView/commit/3d911232d37ab221ecf2ce9a06b27fb960f7934b reinserted strong and removed weak from the ivar. Only the declaration survived your merge ;)